src/main/java/com/evokly/kafka/connect/mqtt/MqttSourceConnectorConfig.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright 2016 Evokly S.A.
+ * See LICENSE file for License
+ **/
+
+package com.evokly.kafka.connect.mqtt;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * MqttSourceConnectorConfig is responsible for correct configuration management.
+ */
+public class MqttSourceConnectorConfig {
+    private static final Logger log = LoggerFactory.getLogger(MqttSourceConnector.class);
+
+    List<Map<String, String>> mConfigs = new ArrayList<>();
+    Map<String, String> mProperties;
+
+    /**
+     * Transform process properties.
+     *
+     * @param properties associative array with properties to be process
+     */
+    public MqttSourceConnectorConfig(Map<String, String> properties) {
+        log.info("Initialize transform process properties");
+
+        mProperties = properties;
+
+        Integer size = Integer.valueOf(properties.get(MqttSourceConstant.CONNECTIONS));
+
+        for (int i = 0; i < size; i++) {
+            Map<String, String> config = new HashMap<>();
+            mConfigs.add(i, config);
+
+            processProperty(i, MqttSourceConstant.KAFKA_TOPIC);
+            processProperty(i, MqttSourceConstant.MQTT_BROKER_URLS);
+            processProperty(i, MqttSourceConstant.MQTT_TOPIC);
+            processProperty(i, MqttSourceConstant.MQTT_QUALITY_OF_SERVICE);
+        }
+    }
+
+    /**
+     * Process property and save to configuration.
+     *
+     * @param index connection number
+     * @param key   field name
+     */
+    private void processProperty(Integer index, String key) {
+        log.info("Process property {}[{}]", key, index);
```

Comment:
imho should be .debug/.trace as this is not "lifecycle" method... only "debugging stuff"